lib/validators/options_processor/options/boolean.js

```diff
@@ -0,0 +1,32 @@
+const MATCH_NOT_FOUND_ERROR = 'Failed to run the test because \'match\' is not provided for \'boolean\' option. Please check README for more information about configuration'
+const UNKNOWN_INPUT_TYPE_ERROR = 'Input type invalid, expected strings "true" or "false", or boolean literal `true` or `false` as input'
+
+class BooleanMatch {
```

Comment:
Since the `boolean` option is added under `validators`, could you please write a test case in one of the validator using `boolean`? just to ensure that the `boolean` option also work for `validators`, not just `payload`